base/scripts/lib/install_fonts.sh

```diff
@@ -0,0 +1,7 @@
+# Install additional fonts
+
+set -e
+apt-get update -y
+apt-get install -y \
+    fonts-noto \
+    fonts-noto-cjk
```

Comment:
Just an empty line missing at the EOF.

```suggestion
    fonts-noto-cjk

```